dotcom-rendering/src/web/components/ContainerLayout.tsx

```diff
@@ -121,6 +125,9 @@ export const ContainerLayout = ({
 		padded={padSides}
 		borderColour={borderColour}
 		backgroundColour={backgroundColour}
+		element="section"
```

Comment:
`section` is the element type on Frontend. It seemed right so I’ve changed this for now.

Reply:
Yeah, `section` is the way to go here. It's very sectiony